CHANGELOG.md

```diff
@@ -41,9 +45,18 @@ Current
 
 
 ### Changed:
+<<<<<<< Updated upstream
 -  [The `TestBinderFactory` now uses the `TestAsynchronousWorkflowsBuilder`](http://github.com/yahoo/fili/pull/39)
     * This allows the asynchronous functional tests to add countdown latches to the workflows where necessary, allowing
       for thread-safe tests.
+=======
+-  [HashPreResponseStore moved to `test` root directory.](https://github.com/yahoo/fili/pull/39)
+   * The `HashPreResponseStore` is really intended only for testing, and does not have capabilities (i.e. TTL) that are needed for production.
+
+- [Removed `JobsApiRequest::handleBroadcastChannelNotification`](https://github.com/yahoo/fili/pull/39)
+   * That logic does not really belong in the `JobsApiRequest` (which is responsible for modeling a response, not processing it), and has
+        been consolidated into the `JobsServlet`.
+>>>>>>> Stashed changes
```

Comment:
Apparently (vanilla) git says nothing if it has conflicts when applying a stash. This is annoying.